src/components2/MapLegend/index.ts

```diff
@@ -0,0 +1,3 @@
+import Legend from './Legend';
+
+export default Legend;
```

Comment:
Der Default Export steht ja schon in https://github.com/FixMyBerlin/fixmy.frontend/pull/960/files#diff-51e051585b938fd97c3dfca3dc3bc0d354ee55b23cebd253b4acd852b3fc8a4bR125.

Ich denken in den barrel files sollten wir - bei neuen components - die named exports verwenden, so dass wir sie darüber auch importieren können.

```suggestion
export { Legend } from './Legend';
```